packages/langium/test/workspace/document-builder.test.ts

```diff
@@ -417,6 +417,25 @@ describe('DefaultDocumentBuilder', () => {
         await builder.build([document], { validation: true });
     });
 
+    test.only('`waitUntil` will correctly resolve if the document is already in the target state.', async () => {
```

Comment:
This fails the CI.

```suggestion
    test('`waitUntil` will correctly resolve if the document is already in the target state.', async () => {
```

Reply:
Whoops, that's a bad habit of mine. I'm glad that the CI catches it as a bad idea.